tools/harness-simulation/README.md

```diff
@@ -10,67 +10,45 @@ Platform developers should modify the THCI implementation and/or the SI implemen
 
 ## POSIX Environment Setup
 
-1. Build OpenThread to generate standalone OpenThread simulation `ot-cli-ftd`. For example, to run OpenThread 1.2 test cases, run the following command in the top directory of OpenThread:
+1. Open the JSON format configuration file `tools/harness-simulation/posix/simulation.conf`:
 
-   ```bash
-   $ CFLAGS='-DOPENTHREAD_CONFIG_IP6_MAX_EXT_MCAST_ADDRS=8' \
-     CXXFLAGS='-DOPENTHREAD_CONFIG_IP6_MAX_EXT_MCAST_ADDRS=8' \
-     script/cmake-build simulation \
-         -DOT_THREAD_VERSION=1.2 \
-         -DOT_DUA=ON \
-         -DOT_MLR=ON \
-         -DOT_COMMISSIONER=ON \
-         -DOT_CSL_RECEIVER=ON \
-         -DOT_SIMULATION_MAX_NETWORK_SIZE=64
-   ```
-
-   Then `ot-cli-ftd` is built in the directory `build/simulation/examples/apps/cli/`.
-
-2. Open the configuration file `config.py`:
-
-   - Edit the value of `OT_PATH` to the absolute path where the top directory of the OpenThread repository is located. For example, change the value of `OT_PATH` to `/home/<username>/repo/openthread`.
+   - Edit the value of `ot_path` to the absolute path where the top directory of the OpenThread repository is located. For example, change the value of `ot_path` to `/home/<username>/repo/openthread`.
+   - As to each entry in `ot_build.ot`,
```

Comment:
I don't think this needs to be split into two bullets, just one should be fine:

- For each entry in `ot_build.ot`, update the value of `number` to be the number of OT FTD simulations needed with the corresponding version.